apps/internal/oauth/ops/accesstokens/tokens.go

```diff
@@ -170,13 +170,14 @@ type TokenResponse struct {
 	RefreshToken string `json:"refresh_token"`
 	TokenType    string `json:"token_type"`
 
-	FamilyID       string                    `json:"foci"`
-	IDToken        IDToken                   `json:"id_token"`
-	ClientInfo     ClientInfo                `json:"client_info"`
-	ExpiresOn      internalTime.DurationTime `json:"expires_in"`
-	ExtExpiresOn   internalTime.DurationTime `json:"ext_expires_in"`
-	GrantedScopes  Scopes                    `json:"scope"`
-	DeclinedScopes []string                  // This is derived
+	FamilyID            string                    `json:"foci"`
+	IDToken             IDToken                   `json:"id_token"`
+	ClientInfo          ClientInfo                `json:"client_info"`
+	ExpiresOnCalculated internalTime.DurationTime `json:"expires_in,omitempty"`
```

Comment:
This exposes the `expires_in` vs. `expires_on` distinction to consumers and makes them responsible for choosing which value to use. The type is then harder to use correctly and the extra complexity has no benefit--consumers just want to know when the token expires, they don't care how that was determined. Why not use `UnmarshalJSON` to set `ExpiresOn`, like you were before?